examples/extract_data/src/extract_data_login.py

```diff
@@ -0,0 +1,136 @@
+IS_LOGGED_IN_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "isLoggedIn": {
+            "type": "boolean",
+            "description": "Use this field to indicate whether the user is logged in."
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem."
+        },
+    },
+}
+TARGET_URL = 'https://www.glassdoor.com/Job/san-francisco-ca-software-engineer-jobs-SRCH_IL.0,16_IC1147401_KO17,34.htm'
+EXTRACT_DATA_PROMPT = "This browser is open to a page that lists available job roles for software engineers in San Francisco. Please provide 10 job roles that appear to be posted by the AI-related companies."
+EXTRACT_DATA_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "companies": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "companyName": {"type": "string"},
+                    "jobTitle": {"type": "string"},
+                    "location": {"type": "string"},
+                    "salary": {
+                        "type": "object",
+                        "properties": {
+                            "min": {"type": "number", "minimum": 0},
+                            "max": {"type": "number", "minimum": 0},
+                        },
+                        "required": ["min", "max"],
+                    },
+                },
+                "required": ["companyName", "jobTitle", "location", "salary"],
+            },
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem.",
+        },
+    },
+}
+
+async def run():
+    try:
+        if not AIRTOP_API_KEY:
+            raise ValueError("AIRTOP_API_KEY is not set")
+
+        client = Airtop(api_key=AIRTOP_API_KEY)
+
+        profile_id = input("Enter a profileId (or press Enter to skip): ").strip()
+        if profile_id:
+            print(f"Using profileId: {profile_id}")
+        else:
+            print("No profileId provided")
+            profile_id = None
+
+        create_session_response = await client.sessions.create({
+            "configuration": {
+                "timeoutMinutes": 10,
+                "persistProfile": not profile_id,
+                "baseProfileId": profile_id,
+            }
+        })
+
+        session = create_session_response["data"]
+        print("Created airtop session", session["id"])
+
+        if not session.get("cdpWsUrl"):
+            raise ValueError("Unable to get cdp url")
+
+        # Create a new window and navigate to the URL
+        window_response = await client.windows.create(session["id"], {"url": LOGIN_URL})
```

Comment:
This is throwing this error:
<img width="1005" alt="image" src="https://github.com/user-attachments/assets/914bd45f-2a08-4dc1-865a-ef135077f74b">

This fixes that:
```
        window = await client.windows.create(session_id, url=LOGIN_URL)
        if not window.data:
            raise Exception("Failed to create window")

        # Get the window ID
        window_id = window.data.window_id

        window_info = await client.windows.get_window_info(session_id, window_id)
```